runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/resources/PipelineResources.java

```diff
@@ -64,6 +70,31 @@
     return resourcePath -> !resourcePath.contains("gradle/wrapper");
   }
 
+  /**
+   * Goes through the list of files that need to be staged on runner. Removes nonexistent
```

Comment:
Don't we error on nonexistent directories, rather than removing them?

Reply:
You are right, documentation updated.